app/hooks/contracts.ts

```diff
@@ -165,7 +170,15 @@ export const useClearingHouse = () => {
         toDecimalStruct(utils.parseUnits(baseAssetAmountLimit)),
         gasLimit
       );
-      await result.wait();
+      const confirmed = await result.wait();
+      enqueueSnackbar({
+        title: "Success",
+        description: "Position creation was successfull",
+        severity: "success",
+        url: network.etherscanLink
+          ? `${network.etherscanLink}block/${confirmed.blockHash}`
```

Comment:
Replace with `${network.etherscanLink}tx/${confirmed.transactionHash}`, that's going to be more useful to the user than the entire block.